content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Connect your database'
 metaTitle: 'Connect your database'
 metaDescription: 'Connect your database to your project'
 langSwitcher: ['typescript', 'node']
-dbSwitcher: ['postgres', 'mysql', 'sqlserver']
+dbSwitcher: ['postgres', 'mysql', 'sqlserver', 'planetscale']
```

Comment:
This page is also missing the "Before" and "Next" buttons at the bottom for PlanetScale (MySQL they are "Installation" and "Using Prisma Migrate")

Reply:
Done